src/app/static/src/js/components/workflowWizard/workflowWizard.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+    <div id="workflow-wizard" class="container">
+        <step v-for="step in steps"
+              :key="getCurrentIndex(step.name)"
+              :toggle-next="toggleNext"
+              :active="isActive(step.name)"
+              :hasVisibility="handleVisibility(step.name)"
+              :valid="validStep"
+              @back="back(step.name)"
+              @next="next(step.name)"
+              @cancel="confirmCancel">
+            <component :is="step.component"
```

Comment:
Very nice!

Reply:
I created a new wizard from scratch.  This, we hope to re-use in our other apps. The [slot](https://vuejs.org/v2/guide/components-slots.html) is a content distribution API which I have used to add another component "step" into the primary component "workflowWizard".